### PR TITLE
inspect code and fix issues

### PR DESCRIPTION
**Pull request by BorisBorshevsky**

### CODE REVIEW

File: config/ini.go [hunk 1]
```diff
@@ -29,15 +29,15 @@ import (
...
-	lineBreak      = "\n"
+	defaultSection  = "default"   // default section means if some ini items not in a section, make them in default section,
+	bNumComment        = []byte{'#'} // number signal
+	bSemComment        = []byte{';'} // semicolon signal
+	bEmpty         []byte
+	bEqual                  = []byte{'='} // equal signal
+	bDQuote                = []byte{'"'} // quote signal
+	sectionStart      = []byte{'['} // section start signal
+	sectionEnd          = []byte{']'} // section end signal
+	lineBreak            = "\n"
```
Comment (astaxie) [inline config/ini.go]: did you format the code?

Comment (BorisBorshevsky) [reply config/ini.go]: I thought I did
Will go through it again in few days to see why everything fails.

Comment (BorisBorshevsky) [reply config/ini.go]: I did, It was not formatted in the first place...

Comment (astaxie) [reply config/ini.go]: but from the diff shows it's not formatted.

Comment (BorisBorshevsky) [reply config/ini.go]: On it

**Comment by BorisBorshevsky** [comment]
I've also ran gofmt on the entire project (hope it is ok...)

The whole PR is based on automatic code analysis tools.

**Comment by flycash** [comment]
This PR is suspended for a long time, I close it

**Pull request closed without merging**